### ISSUE TITLE
Error: "Cannot read property 'forRoot' of undefined" after installing version 2.0.0-alpha.11-1

### ISSUE THREAD

**Issue by aveniri** [angular/material2#2123]
I uninstalled the older version @angular/material@2.0.0-alpha.10 as instructed on the release notes then installed the new one and it doesn't seem to load it since on @NgModule({imports: [ ... ]})
I switched back to 2.0.0-alpha.10 and it works fine.

#### Which versions of Angular, Material, OS, browsers are affected?
angular: 2.2.3, 
material: 2.0.0-alpha.11-1, 
windows 7

**Comment by JanStureNielsen** [comment]
FWIW, I updated my Material `@angular/material@2.0.0-alpha.10` to `@angular/material@2.0.0-alpha.11` in `package.json`, `rm -rf node_modules`, `npm install`, and ran my application without any issues...on Linux.

**Comment by jelbourn** [comment]
Does someone have a deeper stack trace?

**Comment by jelbourn** [comment]
Nevermind, I've reproduced it on a fresh angular-cli project. For some reason my existing project didn't run into this...

**Comment by Toxicable** [comment]
I've tried with alpha.10, alpha11, alpha.11-1 and all of them if you do `console.log(MaterialModule)` it prints  `undefined` importing it with `import { MaterialModule} from '@angular/material';`
I got a repo here with a clean reproduction with angular cli https://github.com/Toxicable/materialtest

**Comment by blyndcide** [comment]
Uncaught TypeError: Cannot read property 'forRoot' of undefined
    at app.module.ts:111
    at Object.<anonymous> (app.module.ts:119)
    at __webpack_require__ (bootstrap 98affba…:52)
    at Object.<anonymous> (rxjs-operators.ts:46)
    at __webpack_require__ (bootstrap 98affba…:52)
    at Object.<anonymous> (src async:7)
    at __webpack_require__ (bootstrap 98affba…:52)
    at Object.<anonymous> (zone.js:1613)
    at __webpack_require__ (bootstrap 98affba…:52)
    at webpackJsonpCallback (bootstrap 98affba…:23)
    at main.bundle.js:1

**Comment by jelbourn** [comment]
I've got a reproduction, trying to figure out _why_ it's happening; AFAICT, MaterialModule is totally there.

**Comment by jelbourn** [comment]
Have a tentative fix on the `@experimental` label, validating it now. Would be awesome if someone else here also confirmed.

**Comment by Toxicable** [comment]
That appears to have done it on the @experimental

**Comment by jelbourn** [comment]
Okay, I've bumped `@latest` to `2.0.0-alpha.11-2` which should fix this. 

For anyone wondering what was wrong: 
alpha.11-1 added some re-exports to `index.ts` files for input, tabs, menu, and select using `export *` (to fix a bug with AoT). We haven't had any issues with `export *` before, but adding these seemed to break WebPack's module loading somehow (I have no idea why). I changed the exports to use specific symbols and everything was peachy. 

I'll consult with @hansl (knows more about webpack than I do) tomorrow and figure out _why_ this happened.
